plugins/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/vmware-tanzu/antrea/plugins
+
+go 1.13
+
+require (
+	github.com/vmware-tanzu/antrea v0.7.1
+	github.com/vmware-tanzu/octant v0.13.1
+	k8s.io/apimachinery v0.19.0-alpha.3
+	k8s.io/client-go v0.19.0-alpha.3
+)
+
+replace (
+	github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20200601065543-2c7a62482f16
```

Comment:
maybe add a comment to the corresponding `replace` directive in the top-level go.mod file as a reminder that this one needs to be kept in-sync

Reply:
OK

Reply:
I guess this is generated automatically after requiring "github.com/vmware-tanzu/antrea"?

Reply:
Is this generated automatically (I thought `replace` statements in dependencies were ignored)? And is it updated automatically when running `go mod tidy`?

Reply:
Actually, I added this line manually because I got compile issue like "github.com/contiv/ofnet@v0.0.0-00010101000000-000000000000: invalid version" without replacement.

Reply:
I didn't try that and was guessing, then I must be wrong.
If you change github.com/contiv/ofnet to v0.0.0 in the root go.mod, do you still need repeating it here? I guess it would import the original "contiv/ofnet" but it doesn't matter for octant. So do client-go.

Reply:
I have tried to change github.com/contiv/ofnet to v0.0.0 in the root go.mod. It will require v0.0.0 ofnet and reported the unknown version issue. I think we need a replace clause here no matter we replace it to the original one or Wenying's repository.